### PR TITLE
qwen docs + new config

### PR DESCRIPTION
**Pull request by ved1beta**
examples + docs 
image

## Release Notes

* **New Features**
  * Added new fine-tuning configuration examples for Qwen3.5 models (27B text-only and 9B multimodal)

* **Documentation**
  * Updated README with fine-tuning configurations, peak VRAM requirements, and new training commands
  * Expanded getting started section with full fine-tuning guidance and memory considerations

### CODE REVIEW

File: examples/qwen3.5/27b-fft.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      content: value
+val_set_size: 0.0
+output_dir: ./outputs/out
+dataset_prepared_path: last_run_prepared
+
+sequence_len: 2048
+sample_packing: true
+
+# Freeze the vision encoder; train only the language model.
+# model.visual.* parameters have requires_grad set to False via gradient hooks.
```
Comment (NanoCode012) [inline examples/qwen3.5/27b-fft.yaml]:
```suggestion
```

File: examples/qwen3.5/27b-fft.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# hub_model_id: username/custom_model_name
+
+# Full fine-tune (FFT) of the text-only path of Qwen3.5-27B.
+#
+# Qwen3.5-27B is an early-fusion VLM (Qwen3_5ForConditionalGeneration), so axolotl
+# loads the full model including the vision encoder. unfrozen_parameters restricts
+# gradient updates to the language model only, freezing model.visual.* and saving
+# the optimizer state that would otherwise be wasted on vision parameters.
+#
+# For multimodal (image+text) fine-tuning, see 7b-lora-vision.yaml.
```
Comment (NanoCode012) [inline examples/qwen3.5/27b-fft.yaml]: Let's not put this here?

File: examples/qwen3.5/9b-fft-vision.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+base_model: Qwen/Qwen3.5-9B
+processor_type: AutoProcessor
+
+# Full fine-tune (FFT) of Qwen3.5-9B with image+text (multimodal) data.
+# Designed for a single 80GB GPU (A100/H100).
+
+# Memory estimate (bf16): ~14 GB weights + ~7 GB 8-bit Adam = ~21 GB base,
+# leaving ample room for activations at sequence_len 4096.
```
Comment (NanoCode012) [inline examples/qwen3.5/9b-fft-vision.yaml]: Remove

**Review by NanoCode012 — COMMENTED** [review]
Could you also check if there's any other places you can reduce comments/ simplify ?

File: docs/multimodal.qmd [hunk 4]
```diff
@@ -184,6 +185,54 @@ base_model: Qwen/Qwen3-VL-4B-Instruct
...
+```
+
+Measured peak VRAM — Qwen3.5-27B, `adamw_bnb_8bit`, `sequence_len: 2048`:
+
+| Metric | Value |
+|---|---|
+| Max active | 52.89 GiB |
+| Device reserved | 53.31 GiB |
+
+See [examples/qwen3.5/27b-fft.yaml](https://github.com/axolotl-ai/axolotl/blob/main/examples/qwen3.5/27b-fft.yaml).
```
Comment (NanoCode012) [inline docs/multimodal.qmd]: These extra details and parts below should be in the README.md and not here

File: examples/qwen3.5/27b-fft.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
+base_model: Qwen/Qwen3.5-27B
+# Automatically upload checkpoint and final model to HF
+# hub_model_id: username/custom_model_name
+
+# Full fine-tune (FFT) of the text-only path of Qwen3.5-27B.
+#
+# Qwen3.5-27B is early-fusion VLM.
+# To freeze vision encoder for text-only training:
+# For multimodal (image+text) fine-tuning, see 9b-lora-vision.yaml.
```
Comment (NanoCode012) [inline examples/qwen3.5/27b-fft.yaml]: Let's not put this here? Let's just explain on readme.

File: examples/qwen3.5/9b-lora-vision.yaml [hunk 6]
```diff
@@ -1,9 +1,9 @@
-base_model: Qwen/Qwen3.5-7B
+base_model: Qwen/Qwen3.5-9B
 processor_type: AutoProcessor
 
-# Qwen3.5-7B and above are early-fusion VLMs (Qwen3_5ForConditionalGeneration).
+# Qwen3.5-9B and above are early-fusion VLMs (Qwen3_5ForConditionalGeneration).
 # Vision and text tokens are processed together by the same transformer layers.
-# Note: Qwen3.5-2B is a text-only model — the smallest VLM is Qwen3.5-7B.
+# Note: Qwen3.5-2B is a text-only model — the smallest VLM is Qwen3.5-9B.
```
Comment (NanoCode012) [inline examples/qwen3.5/9b-lora-vision.yaml]: Simplify/Remove

File: docs/multimodal.qmd [hunk 7]
```diff
@@ -191,6 +192,23 @@ base_model: Qwen/Qwen3-VL-4B-Instruct
 chat_template: qwen2_vl  # same as qwen2-vl
 ```
 
+### Qwen3.5 {#sec-qwen3-5}
+
+::: {.callout-note}
+Qwen3.5-2B is text-only. All larger variants (7B and above) are early-fusion VLMs
+(`Qwen3_5ForConditionalGeneration`) — vision and text tokens share the same transformer
+layers. Axolotl always loads the full VLM class regardless of whether images are present
+in the dataset.
```
Comment (NanoCode012) [inline docs/multimodal.qmd]:
```suggestion
Qwen3.5-2B is text-only. All larger variants are
early-fusion VLMs (`Qwen3_5ForConditionalGeneration`).
```

File: docs/multimodal.qmd [hunk 8]
```diff
@@ -191,6 +192,23 @@ base_model: Qwen/Qwen3-VL-4B-Instruct
...
+:::
+
+```yaml
+base_model: Qwen/Qwen3.5-9B  # smallest VLM variant
```
Comment (NanoCode012) [inline docs/multimodal.qmd]:
```suggestion
base_model: Qwen/Qwen3.5-9B
```

File: examples/qwen3.5/9b-fft-vision.yaml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    split: train[:1%]
+
+dataset_prepared_path: last_run_prepared
+val_set_size: 0.0
+output_dir: ./outputs/out
+
+sequence_len: 4096
+pad_to_sequence_len: false
+
+wandb_project: 9b-vision
```
Comment (NanoCode012) [inline examples/qwen3.5/9b-fft-vision.yaml]:
```suggestion
wandb_project:
```

**Review by NanoCode012 — APPROVED** [review]
